### DISCUSSION TITLE
Aspire + Angular FE?

### DISCUSSION THREAD

**Discussion by 29haspsfan** [NikiforovAll/keycloak-authorization-services-dotnet#138] [Q&A]
I love your repo, and am super excited for you to have it included in the Aspire stack!! You must be pretty stoked that your hard work has been noticed :)

I would typically post this on a discussion section, but didn’t see it available, only issues, and this is definitely not an issue. I was wondering if you had an example where you integrated Angular as the FE and an Aspire project for the rest with vertical slice architecture. I forked this repo and have customized it but really want to use your libraries/packages to handle Keycloak integration. Do you think you could help me out and use it as an example to integrate with? https://github.com/baratgabor/MyWarehouse

**Comment by NikiforovAll** [comment]
Thank you for the feedback. I'm on vacation by the end of August. I will come back to you so we can work on some example and maybe full fledged template.

**Reply by NikiforovAll** [reply]
Hi @CEduChrOCOlOugep, could you please reach me out on discord?

**Reply by 29haspsfan** [reply]
I don't have Discord. Is there another way to talk?
